common/component.go

```diff
@@ -230,7 +245,8 @@ func (c *VcapComponent) ListenAndServe() {
 	}
 
 	f := func(user, password string) bool {
-		return user == c.Varz.Credentials[0] && password == c.Varz.Credentials[1]
+		return (user == c.Varz.Credentials[0] && password == c.Varz.Credentials[1]) ||
+			(user == brokerCredentials.Username && password == brokerCredentials.Password)
 	}
```

Comment:
We already have a host defined as cfg.status.host since we are piggy backing we may want to change the name to something more generic.